benchmarks/video-analytics/go.mod

```diff
@@ -3,23 +3,25 @@ module tests/video_analytics
 go 1.16
```

Comment:
not updated here

Reply:
this did not have that error.
Shall I update this as well?

Reply:
let's update consistently